src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1521,6 +1529,7 @@ void goto_instrument_parse_optionst::help()
     " --model-argc-argv <n>        model up to <n> command line arguments\n"
     // NOLINTNEXTLINE(whitespace/line_length)
     " --remove-function-body <f>   remove the implementation of function <f> (may be repeated)\n"
+    " --block-function <f>         replace body of function <f> with assume(false), effectively block all paths through function\n" // NOLINT(*)
```

Comment:
Consider adding `(may be repeated)` as above to indicate it can be used more than once.